backend/src/youtube-feed/youtube-feed.service.ts

```diff
@@ -30,20 +38,32 @@ export class YouTubeFeedService {
     }
 
     onModuleInit() {
-        this.fetchAndSaveYouTubeFeed();
+        setInterval(() => {
```

Comment:
Use webhook or subscriber API.
They have a subscriber API that basically notify you if this channel has new vid

Reply:
This is the api:
[PUBSUB API](https://developers.google.com/youtube/v3/guides/push_notifications)